build_tools/ci/run_matmul_test.sh

```diff
@@ -460,19 +459,20 @@ function run_matmul_test() {
       --device=${device} \
       --max_elements_to_check=${max_elements_to_check}"
 
+  set +e
 
-
-  echo "**** Running '${name}' matmul test ${num_repeat_runs} times ****"
+  echo "**** Running '${name}' matmul test ${num_repeat_runs} times (command ${COMMAND}) ****"
   for i in $(seq 1 $num_repeat_runs); do
-    echo "Run number ${i} / ${num_repeat_runs} of command ${COMMAND}"
+    bash $THIS_DIR/reset_npu.sh
```

Comment:
@makslevental What's the point of resetting the NPU every run? This would reduce testing coverage it wouldn't catch issues by the AIE/NPU being left in a bad state by some run.